public/app/features/alerting/unified/components/rule-editor/DetailsStep.tsx

```diff
@@ -21,7 +21,11 @@ const recordingRuleNameValidationPattern = {
   value: /^[a-zA-Z_:][a-zA-Z0-9_:]*$/,
 };
 
-export const DetailsStep: FC = () => {
```

Comment:
We could have specified `FC<DetailsStepProps>` but technically this one doesn't render any `children` from props so it's 👌